### ISSUE TITLE
Services are not a good spot for markup

### ISSUE THREAD

**Issue by danfusaro** [MatTaNg/joes-task#8]
Typically a "service" is reserved for network requests, interaction with some external data source, or factory methods and is not typically used for markup/jsx related tasks.  This code should probably be moved to some sort of UI utility.  Of course this is purely semantic, this will work but if this were a proper code review I would make the same comment.

https://github.com/MatTaNg/joes-task/blob/357784dc9116c6ce9696f92ee1070ff28f6f5954/src/services/api.js#L44

**Comment by MatTaNg** [comment]
Yes totally agree, this was due to pure laziness :).  Realistically it should be a functional component.
